plugins/forc-index/src/defaults.rs

```diff
@@ -44,6 +44,7 @@ name = "{indexer_name}"
 version = "0.0.0"
 edition = "2021"
 publish = false
+rust-version = "1.71.0"
```

Comment:
Any reason this is different than the rust-version specified in the workspace manifest? (1.71.1 vs 1.71.0)